### ISSUE TITLE
Jak nakládat s přílohou periodika

### ISSUE THREAD

**Issue by kvetjo** [NLCR/Standard_NDK#200]
Dobrý den,

potřeboval bych poradit s případem zpracování periodika, na který jsme narazili, a se kterým si nevíme moc rady.
Jedná se o "přílohu periodika" (tzn. příloha na úrovni ročníků, společná pro všechny ročníky). Rozumím tomu, že přílohy mohou existovat pouze jakožto "přílohy ročníku" a "přílohy čísla" a že tedy "příloha periodika" není možná. 

Zajímalo by mě jak ale nakládat s případem, kdy bylo celé periodikum (respektive všechny jeho ročníky) svázáno a následně mu byla přidána titulní strana a obsah. Právě obsah nám přijde důležitý a neradi bychom ho ztratili. 

Chtěl bych se tedy zeptat, jestli existuje řešení tohoto problému, nebo jestli je jediná možnost obsah a titulní stranu celého periodika zahodit a příště prostě nedigitalizovat?

Děkuji moc,

S pozdravem
Jonáš Květ
Ústřední knihovna Univerzity Karlovy

**Comment by luckajirku** [comment]
Nejsem teď schopná si vzpomenout, u kterého periodika to bylo, ale měli jsme u něčeho obsah, který vyšel později pro několik ročníků zpětně - a tam jsme ho zduplikovali a dali ten stejný do obou/všech jako přílohu ročníku. Pro uživatele nejlepší řešení, pro Krameria bezproblémové:-).

**Comment by kvetjo** [comment]
Děkuji oběma za odpovědi. Souhlasím s @zabak, že vložit stránky přímo pod úroveň "periodikum" je asi logicky nejsprávnější řešení. Nicméně např. u obsahů pro celý ročník (tzn. např. společný obsah pro všechny čísla) je z nějakého důvodu zavedená praxe vytvářet těmto speciálním stranám objekt příloha, která se pak umístí na úroveň ostatních čísel. Například zde:
[https://ndk.cz/uuid/uuid:efb3ede2-885a-4241-91e6-d11d38a13590](https://ndk.cz/uuid/uuid:efb3ede2-885a-4241-91e6-d11d38a13590) nebo [https://kramerius.lib.cas.cz/uuid/uuid:51258890-b76d-44b8-8bf0-91db24ef81c6](https://kramerius.lib.cas.cz/uuid/uuid:51258890-b76d-44b8-8bf0-91db24ef81c6)
Analogicky by se pak mohla přenést tato praxe i o úroveň výš (a tedy mít k dispozici něco jako "příloha periodika"). Myslím si, že tak by to byla menší změna pro klienta Krameria (stačilo by pouze dovolit tu vazbu, a tedy nevytvářet nový způsob zobrazení). 

Zároveň souhlasím s @luckajirku, že Vámi nabízená rada je asi v současné chvíli nejlepší způsob řešení. Pokud však periodikum bude mít velké množství ročníku, tak to není bohužel nejoptimálnější. Asi to ale nyní uděláme podle vás. 

Díky moc!

**Comment by kvetjo** [comment]
Ano, má to ukončené vydávání. Je to jen 10 ročníků a bylo to dokonce svázané na dvakrát, tzn. 1896 - 1900 a pak následně s novou titulní stranou a obsahem 1901 - 1905. Takže v tomto konkrétním případě to není problém dát to ke každému ročníku, jak jste psala výš - spíš si říkám jestli by obecně nestálo zato podat návrh na změnu pro další, jiné případy. Možná by to využily příště i jiné knihovny v podobných případech. Nevím tedy, jestli to není něco proti něčemu, ale to ať posoudí kompetentnější lidé než jsem já, kteří tvoří standardy :-) 

Myslel jsem však, že návrhy se podávají otevřením zde příslušného issue. Můžu se tedy zeptat, jestli nevíte, jak formálně správně by se měl takovýto návrh podat?  

Díky

**Comment by PavlinaKocisova** [comment]
Dobrý den, Váš dotaz jsme v NK zaregistrovali, jenom jsme se ještě nedostali k odpovědi. Formálně nic posílat netřeba. Jinak souhlasím s návrhem p. Jirků, zřejmě to pro Vás v tuto chvíli bude nejlepší řešení.
